In [1]:
import pandas as pd
import scipy.stats as stats


In [2]:
data = pd.read_csv('vv.csv')
data.head()

item   store_price  est_price
0   Blueberries 1 pint 340 g         7.96       4.55
1        Red Seedless Grapes         8.77       7.95
2  Russet Potatoes 10 lb bag         7.96       4.56
3            Broccoli Crowns         2.88       3.82
4            McIntosh Apples         2.48       1.21

In [4]:
store_price = data.store_price
est_price = data.est_price

differences = [store_price[i] - est_price[i] for i in range(len(store_price))]

t_stat, p_value = stats.ttest_rel(store_price, est_price)

t_stat, p_value

(0.17135232044537793, 0.8652245067388556)

Null hypothesis -> no difference between means 

p-val < 0.05 -> sig evidence against null hyp. (ie: there is a diff bw means)
p-val > 0.05 fail to reject H0

### Results: 
p-val = 0.865 > 0.05

Do not reject the null hypothesis - observed difference is likely due to chance and random variation - ie: random noise imposed on the scaling factors 



In [5]:
from sklearn.metrics import mean_squared_error

mean_squared_error(store_price, est_price)

5.529160714285715